emulator/src/components/hdw-nvs/hdw-nvs.c

```diff
@@ -29,6 +30,22 @@
 #define NVS_ENTRY_BYTES      32
 #define NVS_OVERHEAD_ENTRIES 12
 
+//==============================================================================
+// Structs
+//==============================================================================
+
+typedef struct
+{
+    char key[36];
+    uint32_t size;
+    bool isBlob;
+    union
+    {
+        void* blob;
+        int32_t int32;
```

Comment:
I haven't read what this does yet, but on most non-embedded systems an `int32_t` is half the size of a `void*`.

Reply:
Ah, I see. All good.